server/release-files/2021/09/19/release-0096c2e0-2c55-41c5-8ba1-896f0b072f0a.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Should all of these json files be included in git?

Reply:
Thanks. The .gitignore was updated but not the actual files

Reply:
(files now deleted)